### PR TITLE
(Sentinel) sentinel por event check

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

Add POR submission check

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [x] Should publish Docker image

### CODE REVIEW

File: sentinel/pkg/checker/event/app.go [hunk 1]
```diff
@@ -98,12 +119,24 @@ func check(ctx context.Context) {
...
+	porOffset, err := timeSinceLastPorEvent(ctx, PegPorToCheck)
+	if err != nil {
+		log.Error().Err(err).Str("feed", PegPorToCheck.FeedName).Msg("Failed to check peg por")
+		return
+	}
+	log.Debug().Str("POR offset", porOffset.String()).Msg("POR offset")
+	if porOffset > time.Duration(PegPorToCheck.ExpectedInterval)*time.Millisecond+POR_BUFFER {
+		log.Warn().Str("feed", PegPorToCheck.FeedName).Msg(fmt.Sprintf("%s delayed by %s\n", PegPorToCheck.FeedName, porOffset-time.Duration(PegPorToCheck.ExpectedInterval)*time.Millisecond))
+		msg += fmt.Sprintf("%s delayed by %s\n", PegPorToCheck.FeedName, porOffset-time.Duration(PegPorToCheck.ExpectedInterval)*time.Millisecond)
+	}
```
[the remark this answers is not in the document]
Comment (nick-bisonai) [reply sentinel/pkg/checker/event/app.go]: updated from commit [eb9274e](https://github.com/Bisonai/orakl/pull/1639/commits/eb9274ede8a7d4fb1c55c7406591a1bd5acf3a8c)

File: sentinel/pkg/checker/event/app.go [hunk 2]
```diff
@@ -16,8 +16,10 @@ import (
 const AlarmOffset = 3
 
 var FeedsToCheck = []FeedToCheck{}
+var PegPorToCheck = FeedToCheck{}
 var EventCheckInterval time.Duration
 var BUFFER = 1 * time.Second
+var POR_BUFFER = 1 * time.Minute
```
Comment (Intizar-T) [inline sentinel/pkg/checker/event/app.go:22]: how about adding unit to the variable name to ensure better readability and usability. That way we wont have to come back to the variable definition to know the unit: `BUFFER_SEC` and `POR_BUFFER_MIN`

Comment (nick-bisonai) [reply sentinel/pkg/checker/event/app.go:22]: great suggestion, will do

**Review by martinkersner — APPROVED** [review]
LGTM!

**Pull request merged**